src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+.header {
+  min-width: 1024px;
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+img.logo {
+  display: flex;
+  justify-content: center;
```

Comment:
The `img.logo` class uses `display: flex` and `justify-content: center`, which are not applicable to images. Images are not containers, so these properties have no effect. Consider removing them.